research-areas.md

```diff
@@ -4,34 +4,49 @@ title: Research areas
 
 # Overview
 
-Vac has several research areas it is interested in and has or is developing expertise in. Here’s an overview of some of these areas along with motivation and what type of topics are included. Note that a lot of areas are overlapping in terms of specific projects.
+Vac has several research areas it is interested in and has or is developing expertise in.
+Here’s an overview of some of these areas along with motivation and what type of topics are included.
+Note that a lot of areas are overlapping in terms of specific projects.
 
 ## Incentivization
 
-In an open p2p network, nodes have to be incentivized for services provided to the network. Additionally, these should be privacy-preserving and as trust-minimized as possible. This research area includes (but is not limited to): RLN, service credentials, peer reputation, and protocol revenue.
+In an open p2p network, nodes have to be incentivized for services provided to the network.
+Additionally, these should be privacy-preserving and as trust-minimized as possible.
+This research area includes (but is not limited to):
+RLN, service credentials, peer reputation, and protocol revenue.
 
 ## Applied ZK
 
-Applied ZK opens up the door to many new constructs, especially related to privacy. This research area includes things like RLN, Unirep and similar novel constructs, but also things like R&D related to ZKs in general such as Circom, proves on mobile, different proof systems such as Halo 2, etc. Done in close collaboration with other teams in the space.
+Applied ZK opens up the door to many new constructs, especially related to privacy.
+This research area includes things like RLN, Unirep and similar novel constructs,
+but also things like R&D related to ZKs in general such as Circom, proves on mobile, different proof systems such as Halo 2, etc.
+Done in close collaboration with other teams in the space.
 
 ## Privacy/Anonymity
 
-Specifically metadata privacy. This includes anonymity studies and providing a modular suite of protocols with clear trade-offs and threat models.
+Specifically metadata privacy.
+This includes anonymity studies and providing a modular suite of protocols with clear trade-offs and threat models.
+
+[Privacy/Anonymity Roadmap](https://github.com/vacp2p/research/issues/107)
 
 ## Scalability
 
-Making the network scalable in terms of number of nodes, messages supported, discovering of nodes, caching of historical messages etc. Largely focused on Waku right now, but things like scalable group chats also relevant here.
+Making the network scalable in terms of number of nodes, messages supported, discovering of nodes, caching of historical messages etc.
+Largely focused on Waku right now, but things like scalable group chats also relevant here.
```

Comment:
Not necessarily for this PR, but just as a general comment for future revisions. 
I understand it is just an abstract, but it embodies a lot of key research problems which can benefit from more detailed description. For example, in `scalable group chat`, we need to be precise on the problem definition e.g., key management in encrypted group chats, scalable data synchronization in group chats?  
The same holds for the term `caching of historical messages`, there is not enough context for the readers to relate what the system model exactly is and what aspect needs more scalability (or at least what is that we are aiming to make it scalable).

Reply:
>  I understand it is just an abstract, but it embodies a lot of key research problems which can benefit from more detailed description. For example, in scalable group chat, we need to be precise on the problem definition e.g., key management in encrypted group chats, scalable data synchronization in group chats?

basically decentralized MLS, instead of scaling n^2 with participants have other mechanism (ratchet etc) that is a) secure b) minimize coordination role